### PR TITLE
Feature #143 GPGS for Profile

### PR DESCRIPTION
**Pull request by EugeneButusov**
GPGS for Profile was implemented.

Issues:
- I can't commit `gpg.framework` because GitHub block to push files larger than 100MB (gpg.framework has binary file with 117 mbytes).

### CODE REVIEW

File: social-providers/ios-profile-google/SoomlaiOSProfileGoogle/GPGS Categories/Leaderboard+GPGS.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//
+// Created by Eugene Butusov on 03/12/15.
+// Copyright (c) 2015 SOOMLA Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```
Comment (borisspektor) [inline social-providers/ios-profile-google/SoomlaiOSProfileGoogle/GPGS Categories/Leaderboard+GPGS.h]: @EugeneButusov why do we have a space in the folder name?
And why is it Categories?

File: social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m [hunk 2]
```diff
@@ -512,6 +491,84 @@ - (void)invite:(NSString *)inviteMessage dialogTitle:(NSString *)dialogTitle suc
...
+    } else {
+        fail(@"To use GPGS features, please set `enableGameServices = YES` in Google provider initialization parameters.");
+    }
+}
+
+-(void)showLeaderboards {
+    if (_enableGameServices) {
+        [[GPGLauncherController sharedInstance] presentLeaderboardList];
+    } else {
+        NSLog(@"To use GPGS features, please set `enableGameServices = YES` in Google provider initialization parameters.");
```
Comment (borisspektor) [inline social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m]: @EugeneButusov why NSLog and not SoomlaUtil.log?

File: social-providers/ios-profile-google/SoomlaiOSProfileGoogle/GPGS Categories/Leaderboard+GPGS.h (hunk 1 above)
Comment (EugeneButusov) [inline social-providers/ios-profile-google/SoomlaiOSProfileGoogle/GPGS Categories/Leaderboard+GPGS.h]: @borisspektor, ok, I'll rename folder name.

File: social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m (hunk 2 above)
Comment (EugeneButusov) [inline social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m]: @borisspektor, my mistake, forgot about SoomlaUtils.

File: social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m [hunk 3]
```diff
@@ -50,47 +62,12 @@ -(NSArray *)scopes {
...
-    if (originalMethod == nil) {
-        originalMethod = class_getInstanceMethod([UIApplication class], @selector(openURL:));
-        swizzledMethod = class_getInstanceMethod([self class], @selector(openURL:));
-    }
-    method_exchangeImplementations(originalMethod, swizzledMethod);
-    methodReplaced = !methodReplaced; //signalize that implementations was swapped
-    LogDebug(TAG, forward ? @"Method openURL of UIApplication was overrided." : @"Method openURL of UIApplication was re-setted to original.");
-
-}
-
```
Comment (vedi) [inline social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m:84]: @EugeneButusov, is this code not needed anymore?

Comment (EugeneButusov) [inline social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m:84]: @vedi, yes, I transferred SOOMLA Google Plus from GPPSignIn -> GIDSignIn (GPPSignIn is deprecated and GPGS doesn't work with GPPSignIn) and GIDSignIn opens login view inside app (not in Safari).

**Pull request merged**